src/components/InstantSearch.vue

```diff
@@ -103,6 +104,11 @@ export default {
       );
     },
   },
+  created() {
+    if (this.searchClient.addAlgoliaAgent) {
```

Comment:
We should go for `typeof this.searchClient.addAlgoliaAgent === 'function'` to ensure that we can actually call it in the line below ([see on InstantSearch.js](https://github.com/algolia/instantsearch.js/blob/develop/src/lib/InstantSearch.js#L111)).

Reply:
I can't imagine a use case where someone adds that key without it being a function. Was aware of this tradeoff but will change it, since that's what we do in InstantSearch.js